catalog/core/catalog-core-urlresourcereader/src/main/java/ddf/catalog/resource/impl/URLResourceReader.java

```diff
@@ -583,7 +594,8 @@ private boolean validateFilePath(File resourceFilePath) throws IOException {
     return false;
   }
 
-  protected WebClient getWebClient(String uri, Map<String, Serializable> properties) {
+  @VisibleForTesting
+  WebClient getWebClient(String uri, Map<String, Serializable> properties) {
```

Comment:
I get a compilation error when building. This test needs to be updated with the new `getWebClient` method signature: 
https://github.com/codice/ddf/blob/master/catalog/spatial/ogc/spatial-ogc-urlresourcereader/src/test/java/org/codice/ddf/spatial/ogc/catalog/resource/impl/ResourceReaderTest.java#L94-L95